DEVELOPER_GUIDE.md

```diff
@@ -168,6 +168,8 @@ $ poetry run coverage report
 TOTAL 1207 79 93%
 ```
 
+You can run a combination of these by installing [poetry-exec-plugin](https://github.com/keattang/poetry-exec-plugin) once and using the `poetry exec coverage` shortcut.
```

Comment:
I prefer to run `poetry run coverage html`.  Could we also add `poetry exec coverage-html`?